Floofbot/Modules/Fun.cs

```diff
@@ -52,10 +49,33 @@ public async Task AskEightBall([Summary("question")][Remainder] string question)
             };
             Random random = new Random();
             int randomNumber = random.Next(responses.Count);
+            return responses[randomNumber];
+        }
+
+        public static string RespondEightBall(string question)
+        {
+            Regex questionRe = new Regex(@"^.*\?$");
+            if(questionRe.IsMatch(question)) {
+                string response = Floofbot.Modules.Fun.EightBallResponse();
+                if (!string.IsNullOrEmpty(response))
+                {
+                    string reply = $"> {question}";
+                    reply += "\n";
+                    reply += $"{response}";
+                    return reply;
```

Comment:
I think you can just `return $"> {question}\n{response}"` directly, without even having to create the new variable, unless I'm missing something?

Reply:
You're right, but I thought this would be readable, as the internal representation is multiline as well.

Reply:
It actually confuses me more because I have to mentally concat them, but maybe that's just me :3

Reply:
I'll make it a oneliner ;)